apis/opts/config/ulimit_test.go

```diff
@@ -0,0 +1,73 @@
+package config
+
+import (
+	"testing"
+
+	"github.com/docker/go-units"
+)
+
+func TestUlimit_Set(t *testing.T) {
+	type fields struct {
+		values map[string]*units.Ulimit
+	}
+	type args struct {
+		val string
+	}
+	tests := []struct {
```

Comment:
I was curious why not use
```
struct{
                name    string
		fields     values map[string]*units.Ulimit
		args     string
		wantErr bool
	}
```

Reply:
Ok, thank you for the suggestion. Maybe it's clearer by using the way that you suggest.